kyverno.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: kyverno
+  version: 1.9.3
+  epoch: 0
+  description: Kubernetes Native Policy Management CLI
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/kyverno/kyverno/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: dbc29ccb9a54479addc0f1c6c60879a50b9c20e0bd917cd6e0d123636f68e770
```

Comment:
I might recommend doing a git-checkout here instead. It's usually not a big difference, but some builds work better when they have the .git directory available to them, to populate things like build versions.

Reply:
For sure, going to modify it